src/components/Header.tsx

```diff
@@ -41,13 +41,15 @@ const MenuLink = ({
   return (
     <RouteLink
       href={href}
-      color="white"
-      borderBottomWidth={isActive ? '4px' : 'none'}
-      borderColor={isActive ? 'primary.500' : 'none'}
+      color="primaryTextLightmode"
+      borderBottomWidth="4px"
```

Comment:
I'd say this is the problem: we are now always fixing the border-bottom width to 4px, when it should be none if `isActive` is false.

Reply:
This is still pending, right? 😅

Reply:
I'll go make a commit to try your proposal. 😉